pkg/asset/machines/worker.go

```diff
@@ -27,13 +27,19 @@ import (
 )
 
 const (
-	// workerMachineSetFileName is the filename used for the worker MachineSet manifest.
-	workerMachineSetFileName = "99_openshift-cluster-api_worker-machineset.yaml"
+	// workerMachineSetFileName is the format string for constructing the worker MachineSet filenames.
+	workerMachineSetFileName = "99_openshift-cluster-api_worker-machineset-%s.yaml"
 
 	// workerUserDataFileName is the filename used for the worker user-data secret.
 	workerUserDataFileName = "99_openshift-cluster-api_worker-user-data-secret.yaml"
 )
 
+var (
+	workerMachineSetFileNamePattern = fmt.Sprintf(workerMachineSetFileName, "*")
+
+	_ asset.WritableAsset = (*Worker)(nil)
```

Comment:
this is duplicate of https://github.com/openshift/installer/pull/1481/commits/c0bc4dce154e26f35c3006f785e66f2deb68074e#diff-df3202277ffea159153362ec095a8762R69

Reply:
> this is duplicate of c0bc4dc#diff-df3202277ffea159153362ec095a8762R69

Nice catch.  I've removed the old line to bring `worker.go` closer to `master.go` as part of c0bc4dc -> 009963e.